apps/codeforafrica/src/lib/data/blockify/featuredProjects.js

```diff
@@ -0,0 +1,16 @@
+import { sortTags } from "@/codeforafrica/lib/data/utils/posts";
```

Comment:
Move `sortTags` to `utils/tags`. It's strange for `projects` to import `sortTags` from `posts`.